AppCenterDistribute/AppCenterDistribute/Resources/ru.lproj/AppCenterDistribute.strings

```diff
@@ -15,4 +15,4 @@
 "MSDistributeAppUpdateAvailableMandatoryUpdateMessage" = "Разработчики этого приложения требуют выполнить обновление до %@ %@ (%@).";
 "MSDistributeAppUpdateAvailableOptionalUpdateMessage" = "Обновление %@ %@ (%@) доступно для скачивания и установки.";
 "MSDistributeInAppUpdatesAreDisabled" = "Обновления в приложении отключены.";
-"MSDistributeInstallFailedMessage" = "Эта версия была либо загружена, либо загружена с помощью браузера в частном режиме.";
+"MSDistributeInstallFailedMessage" = "Эта версия либо была загружена не с портала, либо через браузер в приватном режиме.";
```

Comment:
```suggestion
"MSDistributeInstallFailedMessage" = "Эта версия либо не была загружена не с портала, либо загружена через браузер в приватном режиме.";
```

Reply:
Not sure translator game me right translations but suggested one seems better.

Reply:
I agree.